cmd/soroban-rpc/README.md

```diff
@@ -0,0 +1,100 @@
+# Soroban-RPC
+
+Soroban-RPC allows you to communicate directly with Soroban via a JSON RPC interface.
+
+For example, you can build an application and have it send a transaction, get ledger and event data or simulate transactions.
+
+Alternatively, you can use one of Soroban's client SDKs such as the js-soroban-client, which will need to communicate with an RPC instance to access the network.
```

Comment:
this line could probably be removed, the focus here is on building/running RPC, rather than getting into downstream usages.